.github/actions/custom-actions/aarambh_production_custom/action.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Aarambh Production release custom action'
+description: 'Sets up environment variables related to non-standard additional features in use in the Aarambh Production environment'
+runs:
+  using: "composite"
+  steps:
+    # Set IWF credentials
+    - name: Set IWF_API_USERNAME
+      uses: "marvinpinto/action-inject-ssm-secrets@latest"
+      with:
+        ssm_parameter: "IWF_API_USERNAME"
+        env_variable_name: "IWF_API_USERNAME"
+    - name: Set IWF_API_PASSWORD
+      uses: "marvinpinto/action-inject-ssm-secrets@latest"
+      with:
+        ssm_parameter: "IWF_API_PASSWORD"
+        env_variable_name: "IWF_API_PASSWORD"
+    - name: Set IWF_API_URL
+      uses: "marvinpinto/action-inject-ssm-secrets@latest"
+      with:
+        ssm_parameter: "IWF_API_URL"
+        env_variable_name: "IWF_API_URL"
+    # - name: Set IWF_API_ENVIRONMENT
```

Comment:
Why are these commented out? Are they not required? I've been adding them but we should delete them from all the workflows that set up CSAM if they aren't required, to stop idiots like me copying them everywhere :-)

Reply:
They default to "test" mode and null country code. For now I left that commented out intentionally but will change prior to merge this, once @Humairaa127 confirms that the same setup is working fine for staging environment.